### ISSUE TITLE
Razberry 7 Pro with Compute Module 4

### ISSUE THREAD

**Issue by fencer** [home-assistant/operating-system#2170]
### Describe the issue you are experiencing

I’m running HassOS 9.0 on RPI 4 CM with USB boot. I have installed on it a Razberry board and everything was working fine.
I just upgraded from Razberry 2 to Razberry 7 Pro but since then I am having issues on boot. HassOS will print “Interrupt” messages infinitely on screen and never boots. Not sure what this is supposed to mean and if it is related tot eh Razberry chip or some other co-incidence.

### What operating system image do you use?

rpi4-64  (Raspberry Pi 4/400 64-bit OS)

### What version of Home Assistant Operating System is installed?

9.0

### Did you upgrade the Operating System.

Yes

### Steps to reproduce the issue

1. Turn PI Off
2. Turn PI On

### Anything in the Supervisor logs that might be useful for us?

```txt
not booted yet
```

### Anything in the Host logs that might be useful for us?

```txt
Not booted yet
```

**Comment by agners** [comment]
What hardware are you using exactly? (what is "RPI 4 CM"?). Where is the "Interrupt" message printed? On the HDMI screen?

**Comment by fencer** [comment]
Hardware is Raspberry PI Compute Module 4 with the Raspberry Pi carrier board. 
Yes message is printed on the HDMI screen

**Comment by agners** [comment]
Hm, this is probably U-Boot which gets interrupted by characters sent by the radio.

What `config.txt` options are you using? Make sure `enable_uart` is set to `0`.

**Comment by fencer** [comment]
I will check about that but it is the same configuration when using Razzberry V2 as i just swapped the modules

**Comment by fastLion** [comment]
> Hm, this is probably U-Boot which gets interrupted by characters sent by the radio.
> 
> What `config.txt` options are you using? Make sure `enable_uart` is set to `0`.

Hi Agners, can you tell me please whee to find the 'config.txt' in the 9.2 build? The '/boot/config.txt' path does not exist. 
Thanks Jens

**Comment by agners** [comment]
> Hi Agners, can you tell me please where to find the 'config.txt' in the 9.2 build? The '/boot/config.txt' path does not exist. Thanks Jens

When you have OS level SSH access or local console, it is at `/mnt/boot/config.txt`.

**Comment by fastLion** [comment]
> > Hi Agners, can you tell me please where to find the 'config.txt' in the 9.2 build? The '/boot/config.txt' path does not exist. Thanks Jens
> 
> When you have OS level SSH access or local console, it is at `/mnt/boot/config.txt`.

Thanks, unfortunately when I look into the "mnt/" folder there is nothing in. I made a fresh install yesterday (32bit) on a Raspi4. Installed the SSH-Plugin and configured it. So, I believe I am on the OS level. The SSH-Plugin is the only one I have installed after the first running of Homeassistant. Any other idea where I could find the "config.txt"?
